android-components/components/service/nimbus/src/test/java/mozilla/components/service/nimbus/messaging/NimbusMessagingStorageTest.kt

```diff
@@ -57,6 +60,7 @@ class NimbusMessagingStorageTest {
     fun setup() {
         MockitoAnnotations.openMocks(this)
         malformedWasReported = false
+        malformedMessageIds.clear()
```

Comment:
Nit: Maybe we could have it inside  a function with [@After](https://junit.org/junit4/javadoc/4.12/org/junit/After.html) maybe `teardown` or `after` :)